userguide/tutorials/getting_started.adoc

```diff
@@ -136,48 +136,176 @@ You can also go to http://<IP>:8080/api.html to explore the KillBill APIs.
 
 === Tomcat
 
-Users familiar with Java technologies can also install Kill Bill directly in their Web container of choice, such as Tomcat. We recommend using Tomcat version 8.5.
+Users familiar with Java technologies can also install Kill Bill directly in the Web container of their choice. We recommend using Tomcat version 8.5. This section covers the instructions for installing killbill in Tomcat. If this approach is followed, you will also need to configure the database manually as explained below.
```

Comment:
For consistency, we always refer to our product as `Kill Bill` (and not `killbill`).

Reply:
Fixed